siliconcompiler/tools/yosys/syn_asic.py

```diff
@@ -84,14 +84,19 @@ def setup_asic(chip):
                  ",".join(['library', mainlib, 'option', 'var', 'yosys_buffer_output']),
                  step=step, index=index)
 
+    library_has_addermap = \
+        chip.valid('library', mainlib, 'option', 'file', 'yosys_addermap') and \
+        chip.get('library', mainlib, 'option', 'file', 'yosys_addermap')
```

Comment:
Should have `step=step, index=index)` I think

Reply:
pernode is never on this parameter (I initially thought so too)

Reply:
Ah, got it. Good to go then!